core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -415,6 +418,11 @@ object SparkEnv extends Logging {
         advertiseAddress, blockManagerPort, numUsableCores, blockManagerMaster.driverEndpoint)
 
     // NB: blockManager is not valid until initialize() is called later.
+    //     SPARK-45762 introduces a change where the ShuffleManager is initialized later
+    //     in the SparkContext and Executor, to allow for custom ShuffleManagers defined
+    //     in user jars. In the executor, the BlockManager uses a lazy val to obtain the
+    //     shuffleManager from the SparkEnv. In the driver, the SparkEnv's shuffleManager
```

Comment:
I think this comment it no longer true. Driver SparkEnv shufflemanager is created after the plugin initialized.

Reply:
Thanks @tgravescs. Handled both comments here: https://github.com/apache/spark/pull/43627/commits/6d002a361ac2c1dfad48ee530766c9b0a605696f